app/models/piece.rb

```diff
@@ -32,4 +32,20 @@ def valid_move?
    
   end
 
+  #Capture_Logic
+  def move_to!(new_row, new_column)
+    @pieces = Game.find(game_id).pieces
+    @pieces.each do |piece|
+      if piece.position_row == new_row && piece.position_column == new_column
+        if piece.color == color
+          #We can't move to a place where our own pieces are!
+          puts "ERROR"
+        else
+          #Setting to nil indicated captured
+          piece.position_row == nil
+          piece.position_column == nil
```

Comment:
I tested this using an rspec test, and changing this code worked:
piece.position_row = nil
piece.position_column = nil
piece.save

change that = instead of == since  == tests if equal. and piece.save ensures that changes to the record is saved to the Piece model. Hope this helps.